src/App.js

```diff
@@ -44,19 +44,19 @@ function App() {
     fetch(URL)
       .then((response) => response.json())
       .then(async (data) => {
-        // first result in spreadsheet is null.
-        // So when retrieving all results, remove first result
-        if (mode == null) {
-          data.shift();
-        }
         setFetchingData(false);
-
-        const [first, ...results] = data;
-        setData([first, ...results]);
+        // first result in spreadsheet is null.
+        data.shift();
+        const [...uniqueVenuesArray] = data.filter((venue, index) => {
```

Comment:
On second thoughts I don't know if a spread operator is required at all here

Reply:
yeah I don't think the spread is necessary